go/vt/vtctl/grpcvtctldserver/server_test.go

```diff
@@ -7580,7 +7579,71 @@ func TestPlannedReparentShard(t *testing.T) {
 					Nanos:   1,
 				},
 			},
-			shouldErr: true,
+			expectedErr: "duration: seconds:-1 nanos:1 is out of range for time.Duration",
```

Comment:
this might be a bit unstable, how about `expectedError error` and then asserting `errors.Is(err, test.expectedErr)`?

Reply:
Why do you think this would be unstable? `require.EqualError` takes in a string as a second argument. I thought it was meant to be used this way?

Reply:
right, but we ever change the exact wording of the error message, this test will break. versus if we just assert on the _type_ of error being returned, it is more stable

Reply:
Oh I understand now! We want to assert on the type of error. I explored this possibility today, but unfortunately that won't work for this case. Because we are returning the error as `fmt.Errorf("duration: %v is out of range for time.Duration", dpb)` it doesn't have a specific type that we can assert on. Plus I want to ensure we error out with the correct message and not with a different message that doesn't make sense. 
I think for now this is fine and if we change the error message, then we should also change the expectations of the test.